### PR TITLE
Create index.md

### PR DESCRIPTION
**Pull request by Saikumarksk**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

- [x] Registration Pull request.

[] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

**Comment by Saikumarksk** [comment]
can anyone confirm the registration

File: content/teams/Electronics CreeD/index.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+type: "team"                                                        
+title: "Electronics Creed"
```
Comment (Uyadav207) [inline content/teams/Electronics CreeD/index.md:3]: remove quotes

Comment (Uyadav207) [reply content/teams/Electronics CreeD/index.md:3]: only here

File: content/teams/Electronics CreeD/index.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+---
+
+## Project's Overview
+
+_**Theme-Name**_ : AtmaNirbhar Bharat
+
+_**Application-Type**_ :   Web Application
+
+_**Idea-Description**_ :   The Web Application is about
```
Comment (Uyadav207) [inline content/teams/Electronics CreeD/index.md:22]: Description is incomplete. Kindly have a check on it.

File: content/teams/Electronics CreeD/index.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+_**Application-Type**_ :   Web Application
+
+_**Idea-Description**_ :   The Web Application is about 
+
+_**Tech-Stack-Used**_ :   Python, Django, HTML, Java Script, CSS
+
+_**GitHub-Link**_ :   https://github.com/Saikumarksk/Code-Camp 
+
+_**Hosted-Link**_ :   https://github.com/Saikumarksk
```
Comment (Uyadav207) [inline content/teams/Electronics CreeD/index.md:28]: Hosted link is not needed now. Leave is empty or put N/A there.
Hosted link can not be your  github link it is to be a website URL or your real application live link

File: content/teams/Electronics CreeD/index.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+---
+
+# Member Details
+
+_**No of members**_ :
```
Comment (Uyadav207) [inline content/teams/Electronics CreeD/index.md:34]: No of team not mentioned. Please do that.

File: content/teams/Electronics CreeD/index.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+_**country**_ :
+ 
+_**Technical-Skills**_ :
+
+_**Specialization**_ :
+
+_**GitHub-ID**_ :  
+
+
```
Comment (Uyadav207) [inline content/teams/Electronics CreeD/index.md:132]: Delete Left over part ....

Do these changes and will merge you.

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

**Comment by Uyadav207** [comment]
@Saikumarksk , PLEASE MAKE CHANGES REQUESTED ASAP...

**Pull request closed without merging**